packages/docs/docs/02-Guides/02-Reading/03-Handling Falsy.mdx

```diff
@@ -0,0 +1,33 @@
+# Handling Falsy types
+
+`useDapp` supports handling `False`
+
+```tsx
+export type Falsy = false | 0 | '' | null | undefined
+```
+
+If you pass one of these types to call, it will return undefined without making blockchain query.
+
+## Example
+
+```tsx
+export function useTokenBalance(
```

Comment:
We are interested in an example from the point of view of the user.
So something like:

```
const {account} = useEthers() // account is of type "string | undefined"
const balance = useEtherBalance(account) // Will start making queries once account is defined
```